src/components/footer/index.tsx

```diff
@@ -0,0 +1,70 @@
+'use client';
+import Image from 'next/image';
+
+import { BiLogoMedium } from 'react-icons/bi';
+import { RiTwitterXLine } from 'react-icons/ri';
+
+import { cn } from '@/lib/classnames';
+
+import { CONTROL_BUTTON_STYLES, CONTROL_ICON_STYLES } from '@/components/map/controls/constants';
```

Comment:
```suggestion
import { CONTROL_BUTTON_STYLES, CONTROL_ICON_STYLES } from '@/components/map/controls/constants';
```

why do you consider re-use the styles from the map?? they are not control buttons